dali/test/python/test_RN50_data_pipeline.py

```diff
@@ -22,11 +22,29 @@
 import time
 
 class CommonPipeline(Pipeline):
-    def __init__(self, batch_size, num_threads, device_id, prefetch, fp16, nhwc):
+    def __init__(self, batch_size, num_threads, device_id, prefetch, fp16, nhwc, decoder_type, cache_size, cache_threshold, cache_type, reader_queue_depth):
```

Comment:
I think that all these new parameters should be grouped into one `decoder_params` with multiple fields (or perhaps just a dictionary).

Reply:
will do